_docs/layout/demo-options.md

```diff
@@ -9,12 +9,12 @@ resource: false
 
```

Comment:
This is a non-public page. No changes needed as this probably will get removed.

Reply:
Changes reversed.

Reply:
One missed, but I'm not going to worry about it.